src/ToDoList.js

```diff
@@ -188,7 +176,7 @@ const ToDoList = () => {
               <ListItemText
                 primary={task.task}
                 primaryTypographyProps={{
```

Comment:
We try not to use `Material UI` way to design its elements.
It's a little bit ugly, but we prefer to override their classnames using the object styles that we define.
Let me know if you didn't understand :)

Reply:
I don't understand, should I use the style prop instead?

Reply:
nope, inside the `containerStyle` template string,
we can define a nested `material ui` element classname in order to style it.
```
const containerStyle = css`
  .someNestedMaterialElementClass {
    background: red;
  }
`
```

Reply:
Not sure if this is what you meant: 
```
const containerStyle = css`
  display: flex;
  justify-content: center;
  align-items: center;
  flex-direction: column;
  gap: 30px;
  .css-o2w69a-MuiTypography-root {
    color: #29b6f6;
    font-size: 50px;
    font-weight: bold;
    margin-top: 30px;
  }
`;
```